.github/actions/compute-artifact-names/action.yaml

```
@@ -3,7 +3,10 @@ description: Central place for computing the artifact names in the CI builds.
 
 inputs:
   build-name:
-    description: The build name (e.g., linux-x86_64, macos-aarch64, etc.)
+    description: >
+      The build name (e.g., linux-x86_64, macos-aarch64, etc.). The special
+      value "all" can be used to receive a glob that matches any or all of
+      the names.
```

```suggestion
      The build name (e.g., linux-x86_64, macos-aarch64, etc.). The special
      value "all" can be used to receive a glob that matches all of the build
      names.
```

I got confused by `any or all`